.gitignore

```diff
@@ -3,6 +3,9 @@ out/
 !**/src/main/**/out/
 !**/src/test/**/out/
 .idea/
+pom.xml
```

Comment:
this one should in git)) here we have configs

Reply:
added pom file to git